src/segment-loader.js

```diff
@@ -2129,25 +2129,23 @@ export default class SegmentLoader extends videojs.EventTarget {
     // before retrying.
     const timeToRemoveUntil = currentTime - MIN_BACK_BUFFER;
 
-    this.logger_(`On QUOTA_EXCEEDED_ERR, removing video from 0 to ${timeToRemoveUntil}`);
-    this.sourceUpdater_.removeVideo(0, timeToRemoveUntil, () => {
-      this.logger_(`On QUOTA_EXCEEDED_ERR, removing audio from 0 to ${timeToRemoveUntil}`);
-      this.sourceUpdater_.removeAudio(0, timeToRemoveUntil, () => {
-        this.logger_(`On QUOTA_EXCEEDED_ERR, retrying append in ${MIN_BACK_BUFFER}s`);
-        this.waitingOnRemove_ = false;
-        // wait the length of time alotted in the back buffer to prevent wasted
-        // attempts (since we can't clear less than the minimum)
-        this.quotaExceededErrorRetryTimeout_ = window.setTimeout(() => {
-          this.logger_('On QUOTA_EXCEEDED_ERR, re-processing call queue');
-          this.quotaExceededErrorRetryTimeout_ = null;
-          this.processCallQueue_();
-        }, MIN_BACK_BUFFER * 1000);
-      });
-    });
+    this.logger_(`On QUOTA_EXCEEDED_ERR, removing audio/video from 0 to ${timeToRemoveUntil}`);
+    this.remove(0, timeToRemoveUntil, () => {
```

Comment:
I think we want to remove everything, and it's much easier to use our `remove` function that already does all of that.